src/main/java/net/bhardy/bizzo/billing/impl/PolicyBuilderImpl.java

```diff
@@ -26,6 +26,8 @@
 import java.time.LocalDate;
 import java.util.ArrayList;
 import java.util.List;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.List'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)